.buildkite/pipeline.yml

```diff
@@ -107,3 +107,70 @@ steps:
         depends_on:
           - "operator-image-build"
           - "e2e-tests-image-build"
+
+  - group: redhat-release
+
+    steps:
+
+      - label: "build operatorhub tool"
+        key: "build-operatorhub-tool"
+        if: build.message == "run operatorhub release"
+        commands:
+           - cd hack/operatorhub
+           - make build
+           - buildkite-agent artifact upload "bin/operator*"
+
+      - label: "Generate CRDs"
+        key: "generate-crds"
+        if: build.message == "run operatorhub release"
+        depends_on:
+          - "build-operatorhub-tool"
+        commands:
+           - git checkout $$OHUB_BRANCH
+           - make generate-crds-v1
+        artifact_paths:
+        - "config/*.yaml"
+
+      - label: "run container push"
+        key: "redhat-container-push"
+        if: build.message == "run operatorhub release"
+        depends_on:
+          - "generate-crds"
+        commands:
+           - buildkite-agent artifact download "bin/operator*" /usr/local/
+           - chmod u+x /usr/local/bin/operatorhub
+           - /usr/local/bin/operatorhub container push --dry-run=false
+
+      - label: "run container preflight"
+        key: "redhat-preflight"
+        if: build.message == "run operatorhub release"
+        depends_on:
+          - "redhat-container-push"
+        commands:
+           - curl -o /tmp/jq https://stedolan.github.io/jq/download/linux64/jq && chmod +x /tmp/jq
```

Comment:
A custom image could also contain `jq`

Reply:
Agreed.  I've opened a PR to add jq to our custom image we already use @pebrc